packages/lesswrong/server/resolvers/elicitPredictions.ts

```diff
@@ -0,0 +1,173 @@
+import { addGraphQLSchema, addGraphQLResolvers, addGraphQLQuery, addGraphQLMutation } from '../../lib/vulcan-lib/graphql';
+import fetch from 'node-fetch'
+import { DatabaseServerSetting } from '../databaseSettings';
+import { generateIdResolverSingle } from '../../lib/utils/schemaUtils';
+import { elicitSourceURL } from '../../lib/publicSettings';
+
+const ElicitUserType = `type ElicitUser {
+  isQuestionCreator: Boolean
+  displayName: String
+  _id: String
+  sourceUserId: String
+  lwUser: User
+}`
+
+addGraphQLSchema(ElicitUserType);
+
+const ElicitPredictionType = `type ElicitPrediction {
+  _id: String
+  predictionId: String
+  prediction: Float
+  createdAt: Date
+  notes: String
+  creator: ElicitUser
+  sourceUrl: String
+  sourceId: String
+  binaryQuestionId: String
+}`
+
+addGraphQLSchema(ElicitPredictionType);
+
+const ElicitBlockDataType = `type ElicitBlockData {
+  _id: String
+  title: String
+  notes: String
+  resolvesBy: Date
+  resolution: Boolean
+  predictions: [ElicitPrediction]
+}`
+
+addGraphQLSchema(ElicitBlockDataType);
+
+const elicitAPIUrl = "https://elicit.org/api/v1"
+const elicitAPIKey = new DatabaseServerSetting('elicitAPIKey', null)
+// const elicitSourceName = new DatabaseServerSetting('elicitSourceName', 'LessWrong')
+
+async function getPredictionsFromElicit(questionId: string) {
+  const response = await fetch(`${elicitAPIUrl}/binary-questions/${questionId}/binary-predictions?user_most_recent=true&expand=creator&prediction.fields=createdAt,notes,id,sourceUrl,sourceId,binaryQuestionId&creator.fields=sourceUserId`, {
```

Comment:
```suggestion
  const response = await fetch(`${elicitAPIUrl}/binary-questions/${questionId}/binary-predictions?${querystring.encode({
    user_most_recent: "true",
    expand: "creator",
    "prediction.fields": "createdAt,notes,id,sourceUrl,sourceId,binaryQuestionId",
    "creator.fields": "sourceUserId"
  })}`, {
```
(`querystring.encode` is a nodejs builtin, import with `import querystring from 'querystring'`).